experimental/examples/falling-balls-mpp/build.gradle.kts

```diff
@@ -156,14 +156,15 @@ compose.experimental {
         projectName = "FallingBalls"
         deployConfigurations {
             simulator("IPhone8") {
-                //Usage: ./gradlew iosDeployIPhone8
+                //Usage: ./gradlew iosDeployIPhone8Debug
```

Comment:
In general, how are `deployConfigurations` and target platforms related? If there is a WatchOs target, how can I add a deploy device for it?

Reply:
Right now WatchOs not available with compose-multiplatform.

In the future, it is expected that it will be possible to specify any configuration, including WatchOs inside the deployConfigurations {

Reply:
Gotcha. There is an assumption that each `DeployTarget` will correspond to exactly one variant defined in a multiplatform project, which is perfectly fine.

(Related to this comment - https://github.com/JetBrains/compose-jb/pull/1944#discussion_r830332841)
I think it is helpful to think about `DeployTarget` as shared resources, used to deploy apps, but also to run tests. In a Gradle build with 100s of subprojects, we probably want to limit how many simulators are launched, so having a build-wide understanding of these resources would be quite nice.